sdk/python/feast/infra/online_stores/dynamodb.py

```diff
@@ -50,10 +51,13 @@ class DynamoDBOnlineStoreConfig(FeastConfigBaseModel):
     """Online store type selector"""
 
     region: StrictStr
-    """ AWS Region Name """
+    """AWS Region Name"""
 
     table_name_template: StrictStr = "{project}.{table_name}"
-    """ DynamoDB table name template """
+    """DynamoDB table name template"""
+
+    sort_response: bool = True
+    """Wether or not to sort BatchGetItem response."""
```

Comment:
typo

Reply:
thanks for reviewing this @woop , what do you mean with?

> remove the training full stop for consistency